ui/src/components/partials/SectionBox.js

```diff
@@ -29,7 +38,7 @@ function SectionBox(props) {
             truncate={true}
             data-tip={emptyMessage}
           >
-            {header}
+            <Anchor onClick={() => props.history.push(path)}>{header}</Anchor>
```

Comment:
Lets pass the `path` into the `Anchor`'s `path` prop, so that its clickable and you can right-click, because with the `onClick` method we can't get the link (its just an event)

We should do the same in the other Anchors of this PR

Reply:
Removed the link from the title. Add the path to the path attribute and removed the onClick method